frontend/__tests__/admin/user-list/user-modal.test.js

```diff
@@ -0,0 +1,73 @@
+import { mutate } from "swr";
+import { shallow } from "enzyme";
+import UserModal from "@frontend/components/admin/user-modal";
+
+global.fetch = jest.fn(() =>
+    Promise.resolve({
+        json: () => Promise.resolve("Something"),
+    })
+);
+
+jest.mock("swr", () => {
+    const originalModule = jest.requireActual("swr");
+    return {
+        __esModule: true,
+        ...originalModule,
+        mutate: jest.fn(),
+    };
+});
+
+const onClose = jest.fn();
+
+describe("user modal component test", () => {
+    it("renders the user modal", () => {
+        //Given
+        const dummyUser = {
+            AccountId: null,
+            FirstName: "firstName",
+            LastName: "lastName",
+            Role: "Patient",
+        };
+        //When
+        let wrapper = shallow(<UserModal userInfo={dummyUser} />);
+        //Then
+        expect(wrapper.find("#user-name").text()).toEqual(`${dummyUser.LastName}, ${dummyUser.FirstName}`);
+    });
+    it("Checks if onChange changes state", () => {
+        //Given
+        const dummyUser = {
```

Comment:
since you're using this dummy user thrice, i  think it could/should be moved to the mock file in the tests folder